openlibrary/templates/books/edition-sort.html

```diff
@@ -64,7 +64,7 @@
           <!-- Formats are store as a raw string -->
           $book.physical_format.replace('[', '').replace(']','')
           $if book.languages:
-            $:_('in %(languagelist)s', languagelist=commify_list(thingrepr(l) for l in book.languages))
+            $:_('in %(languagelist)s', languagelist=commify_list(l.name for l in book.languages))
```

Comment:
the language `l` may need to get wrapped in websafe since we're using `$:`
cc: @cdrini